app/styles/components/project.css

```
@@ -1,28 +1,39 @@
+.project-card__content {
+   flex: 1;
+}
 
 .project-card__title {
   margin: 0;
 }
 
-.project-card__description {
-  opacity: 0.8;
-}
+.project-card__description {}
```

Stylelint (17.5.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)